x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/result_settings_non_text_fields_header.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiTableHeader, EuiTableHeaderCell } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const ResultSettingsNonTextFieldsHeader: React.FC = () => {
+  return (
+    <EuiTableHeader className="resultSettingsTable__nonTextHeader">
```

Comment:
Not a change request unless you feel super strongly, but if we wanted to fix the multiple theads issue we would change this to an `EuiTableRow` instead of `EuiTableHeader`. The children can remain as `EuiTableHeaderCell` elements.

Reply:
This was easy enough to do so I went ahead and did it: https://github.com/elastic/kibana/pull/95462/commits/c19a6c4ec90254704a486c915156aec3ff321e74